pkg/apis/authorization/v1beta1/register.go

```diff
@@ -37,10 +39,15 @@ func AddToScheme(scheme *runtime.Scheme) {
 // Adds the list of known types to api.Scheme.
 func addKnownTypes(scheme *runtime.Scheme) {
 	scheme.AddKnownTypes(SchemeGroupVersion,
+		&v1.ListOptions{},
```

Comment:
I added code in the installer path to force api groups to register common options, because many times people forget to do so. Registering ListOptions for group that doesn't support List doesn't hurt.